### PR TITLE
Add SDK

### PR DESCRIPTION
**Pull request by hinthornw**

### CODE REVIEW

File: libs/checkpoint/langgraph/store/base.py [hunk 1]
```diff
@@ -53,6 +61,15 @@ def __eq__(self, other: object) -> bool:
     def __hash__(self) -> int:
         return hash((self.namespace, self.key))
 
+    def dict(self) -> dict:
```
Comment (hinthornw) [inline libs/checkpoint/langgraph/store/base.py:64]: This is convenient in the server - am fine moving it out

File: libs/sdk-py/langgraph_sdk/client.py [hunk 2]
```diff
@@ -1874,6 +1879,195 @@ async def search(
...
+            None
+
+        Example Usage:
+
+            await client.store.delete_item(
+                ["documents", "user123"],
+                key="item456",
+            )
+        """
+        await self.http.post(
```
Comment (nfcampos) [inline libs/sdk-py/langgraph_sdk/client.py:1963]: Shouldn't this use DELETE method?

File: libs/sdk-py/langgraph_sdk/schema.py [hunk 3]
```diff
@@ -197,6 +197,27 @@ class RunCreate(TypedDict):
     multitask_strategy: Optional[MultitaskStrategy]
 
 
+class Item(TypedDict):
+    namespace: list[str]
+    """The namespace of the item."""
+    id: str
+    """The unique identifier of the item within its namespace."""
+    value: Any
```
Comment (nfcampos) [inline libs/sdk-py/langgraph_sdk/schema.py:205]: Don't we enforce this to be dict?

**Review by nfcampos — APPROVED** [review]

File: libs/sdk-py/langgraph_sdk/schema.py (hunk 3 above)
Comment (hinthornw) [reply libs/sdk-py/langgraph_sdk/schema.py:205]: Ah ya good good point

File: libs/sdk-py/langgraph_sdk/client.py (hunk 2 above)
Comment (hinthornw) [reply libs/sdk-py/langgraph_sdk/client.py]: Updated this and GET and PUT

**Pull request merged**